.github/workflows/CI.yml

```diff
@@ -37,26 +49,45 @@ jobs:
         ctest --output-on-failure
     - name: Kcov
       run: curl -s https://raw.githubusercontent.com/Geode-solutions/actions/master/kcov.sh | bash -s -- ${GITHUB_WORKSPACE}/build/opengeode/bin
+      if: matrix.cc == 'gcc' && github.event_name == 'pull_request'
     - uses: codecov/codecov-action@v1.0.2
+      if: matrix.cc == 'gcc' && github.event_name == 'pull_request'
       with: 
         token: ${{ secrets.CODECOV_TOKEN }}
         file: kcov/kcov-merged/cobertura.xml
+    - name: Notify slack
+      if: failure() && github.ref == 'refs/heads/master'
+      env:
+        SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+      uses: pullreminders/slack-action@master
+      with:
+        args: '{\"channel\":\"CLJ1PS7UN\",\"text\":\"Compilation or Testing failed for OpenGeode on Ubuntu\"}'        
       
   build-mac:
     runs-on: macOS-10.14
     needs: format
+    strategy:
+      matrix:
+        cmake_args: ['-DCMAKE_BUILD_TYPE=Debug', '-G "Xcode"']
 
     steps:
     - uses: actions/checkout@v1
     - name: Compile & Test
       run: |
         mkdir -p build
         cd build
-        cmake -DCMAKE_BUILD_TYPE=Debug ..
-        cmake --build . -- -j2
+        cmake ${{ matrix.cmake_args }} ..
+        cmake --build . --config Debug
```

Comment:
You do not need Debug when you use clang without Xcode. I presume that it does not hurt but is it possible to remove it when it is clang without Xcode (to be cleaner)?

Reply:
I know but I did not find à clean way to do it yet... 😕